src/components/elementView/powerView/powerView.tsx

```diff
@@ -365,7 +373,21 @@ export class PowerView extends React.PureComponent<
   }
 
   private renderNoPowerPortsCallout() {
-    return <Callout title="No power ports" icon={IconNames.INFO_SIGN} />;
+    return (
+      <Callout
+        title="This asset does not have any power ports"
```

Comment:
making this message longer also fixed the margin problem on inline button popup hehe